apps/web/pages/index.vue

```diff
@@ -59,6 +59,33 @@ const { data: categoryTree } = useCategoryTree();
 const recommendedProductsCategoryId = ref('');
 definePageMeta({ pageType: 'static' });
 
+const getDefaultHomepageTemplate = () => {
```

Comment:
**question**: What's your reasoning for creating a getter here instead of storing the object in the constant?